plutus-example/plutus-example.cabal

```diff
@@ -157,3 +158,7 @@ test-suite plutus-example-test
                         Test.PlutusExample.ScriptData
 
   ghc-options:          -threaded -rtsopts -with-rtsopts=-N -with-rtsopts=-T
+
+  build-tool-depends:   cardano-node:cardano-node
+                      , cardano-cli:cardano-cli
```

Comment:
This is required because `cardano-node` and `cardano-cli` need to be available for the tests to invoke.